pkgs/applications/misc/gnss-sdr/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv, fetchFromGitHub
+, armadillo
+, blas
+, boost
+, cmake
+, gfortran
+, glog
+, gmock
+, openssl
+, google-gflags
+, gnuradio
+, liblapack
+, orc
+, pkgconfig
+, pythonPackages
+, uhd
+}:
+
+stdenv.mkDerivation rec {
+  name = "gnss-sdr-${version}";
+  version = "0.0.9";
+
+  src = fetchFromGitHub {
+    owner = "gnss-sdr";
+    repo = "gnss-sdr";
+    rev = "v${version}";
+    sha256 = "0gis932ly3vk7d5qvznffp54pkmbw3m6v60mxjfdj5dd3r7vf975";
+  };
+
+  buildInputs = [
+    armadillo
+    blas
+    boost.dev
+    cmake
+    gfortran.cc.lib
+    glog
+    gmock
+    openssl.dev
+    google-gflags
+    gnuradio
+    liblapack
+    orc
+    pkgconfig
+    pythonPackages.Mako
+    uhd
+  ];
+
+  enableParallelBuilding = true;
+
+  cmakeFlags = [
+    "-DGFlags_ROOT_DIR=${google-gflags}/lib"
+    "-DGLOG_INCLUDE_DIR=${glog}/include"
+  ];
+
+  meta = with stdenv.lib; {
+    description = "An open source Global Navigation Satellite Systems software-defined receiver";
+    homepage = http://gnss-sdr.org/;
+    license = licenses.gpl3Plus;
+    platforms = platforms.unix;
```

Comment:
I'm trying to build this on `darwin`, but at least one of the dependencies requires `linux` currently, so unless we can get it to work on macOS this should be changed to `platforms.linux` in the meantime.